### PR TITLE
Fixes #216 : Age limit fixed.

### LINKED ISSUE #216: App crashes for high numerical values.
The app crashes if we enter high numerical values.
This can be seen in following places
1. `Age` in `UserProfile`.
age
2. `Settings` dialog box in `Medicine Store`.
settings
3.  `Add Medicine to Store` dialog box in `Medicine Store`. (Already reported in issue # 197 )
add medicine

@chhavip @yatna
I would like to work to correct first two.
Also if you could tell me what range should be permitted the above text fields.

### PR DESCRIPTION
**Pull request by free4murad**
Fixes issue #216

@yatna @chhavip @anubhakushwaha 
please review the PR.

### CODE REVIEW

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/UserProfile.java [hunk 1]
```diff
@@ -81,6 +83,32 @@ public void onClick(View view) {
...
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
+                lastAge = s.toString();
+                age = 0;
+            }
+
+            @Override
+            public void onTextChanged(CharSequence s, int start, int before, int count) {
+                if(s.toString().length() > 2){
+                    userAgeEt.setError(getString(R.string.max_two_characters));
```
Comment (anubhakushwaha) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/UserProfile.java]: @free4murad Give a better warning text to users like age limit exceeded

Comment (free4murad) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/UserProfile.java]: Sure

**Comment by anubhakushwaha** [comment]
@free4murad Please look that the third point resolved of #216 has not been resolved in this PR? The other contributor asked for the second point.So please include that third point as well.

**Comment by free4murad** [comment]
@anubhakushwaha 
Fixing the age limit will automatically fix this issue.
Please tell me if I'm getting it right

**Comment by anubhakushwaha** [comment]
@free4murad You are talking about add medicine to store dialog right?

**Comment by free4murad** [comment]
@anubhakushwaha 
Sorry I got confused between point 1 &3 of #216 
Will update the PR

**Comment by free4murad** [comment]
@anubhakushwaha 
I have updated the PR. Please check.

**Pull request merged**